servers/cromwell/jobs/controllers/jobs_controller.py

```diff
@@ -278,7 +277,7 @@ def format_task(task_name, task_metadata):
         execution_status=task_statuses.cromwell_execution_to_api(
             latest_attempt.get('executionStatus')),
         start=_parse_datetime(latest_attempt.get('start'))
-        or datetime.utcnow(),
+        or _parse_datetime(latest_attempt.get('end')) or datetime.utcnow(),
```

Comment:
I'm not following how this addresses missing start times 😞

Reply:
if `start` is not defined, the old behavior would set it at `datetime.utcnow()`, which, if the task failed super quickly, would be after the `end` time, and that would cause issues.

Reply:
Ah okay that makes sense, thanks. Maybe add that brief explanation as a comment?

Reply:
will do